packages/api/src/constants/countries.ts

```diff
@@ -0,0 +1,507 @@
+export const ALPHA_2_TO_ALPHA_3: Record<string, string> = {
+    AF: 'AFG',
```

Comment:
I just copy pasted mappings for country code instead of installing another library, since it takes up less space